src/app/dev/DevToys.Tools/Tools/Converters/JsonTable/JsonTableConverterCommandLineTool.cs

```diff
@@ -0,0 +1,95 @@
+﻿using DevToys.Tools.Helpers;
+using DevToys.Tools.Tools.Converters.JsonTable;
+using Microsoft.Extensions.Logging;
+using OneOf;
+
+namespace DevToys.Tools.Tools.Converters.JsonYaml;
+
+[Export(typeof(ICommandLineTool))]
+[Name("JsonTableConverter")]
+[CommandName(
+    Name = "jsonToTable",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Converters.JsonTable.JsonTableConverter",
+    DescriptionResourceName = nameof(JsonTableConverter.Description))]
+internal sealed class JsonTableConverterCommandLineTool : ICommandLineTool
+{
+    [CommandLineOption(
+        Name = "input",
+        Alias = "i",
+        IsRequired = true,
+        DescriptionResourceName = nameof(JsonTableConverter.InputOptionDescription))]
+    internal OneOf<FileInfo, string>? Input { get; set; }
+
+    [CommandLineOption(
+        Name = "outputFile",
+        Alias = "o",
+        DescriptionResourceName = nameof(JsonTableConverter.OutputFileOptionDescription))]
+    internal FileInfo? OutputFile { get; set; }
+
+    // Default to comma-separated-values because output-to-file is the most likely use-case.
+    [CommandLineOption(
+        Name = "copyFormat",
```

Comment:
The term "copy" is confusing since the CLI won't copy something to the clipboard. How about simply `format` and `f` as an alias?

Reply:
Good idea